base/applications/mspaint/atlimagedx.h

```diff
@@ -7,94 +7,72 @@
 
 #pragma once
 
+#include <atlimage.h>
+
 class CImageDx : public CImage
 {
 public:
-    CImageDx() : CImage()
-    {
-        GetImageHorizontalResolution = NULL;
-        GetImageVerticalResolution = NULL;
-        BitmapSetResolution = NULL;
-    }
+    using CImage::FileTypeFromExtension;
+    using CImage::CImage;
```

Comment:
That one isn't automatic when inheriting CImage above? Otherwise, "class CImageDx : private CImage" ?

Reply:
> That one isn't automatic when inheriting CImage above?

No. See https://stackoverflow.com/questions/347358/inheriting-constructors .

> Otherwise, "class CImageDx : private CImage" ?

It fails to compile.